src/server/components/auth/authenticated-user.ts

```diff
@@ -50,6 +43,38 @@ export default class AuthenticatedUser {
     return lists ?? [];
   }
 
+  static calculateSortOrder(customSortOrder?: Record<string, string>): Array<Record<string, string>> {
+    const defaultSortOrder: Array<Record<string, string>> = [
+      {
+        country: "asc",
+      },
+      {
+        type: "asc",
+      },
+    ];
```

Comment:
I think you should be able to use just an object. 

```suggestion
    const defaultSortOrder = {
      country: "asc",
      type: "asc"
    }
```

then if a user selects their own sort order, we should always still try to order by country/type, so we can do something like 

```
const orderBy = {
  ...defaultSortOrder,
  ...actionNeeded: "asc", // or ...parsedQueryParameters(req.query) for future. 
}
```

so if a bunch of lists have 5 list items with actionNeeded, then country/type default sort is retained. if the user decides to invert country sort, then it will be overridden. 

```
const orderBy = {
  ...defaultSortOrder,
  ...country: "desc"
}
```